test/model/PieceTest.java

```diff
@@ -0,0 +1,49 @@
+package model;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class PieceTest {
+
+    private Piece blackPiece = new Piece(PieceColor.BLACK);
+    private Piece whitePiece = new Piece(PieceColor.WHITE);
+
+    @Test
+    public void testIsBlack() {
+        assertTrue(this.blackPiece.isBlack());
+        assertFalse(this.whitePiece.isBlack());
+    }
+
+    @Test
+    public void testIsNull() {
+        assertFalse(this.blackPiece.isNull());
+    }
+
+    @Test
+    public void testIsSameColorAs() {
+        Piece blackPiece2 = new Piece(PieceColor.BLACK);
+        Piece whitePiece2 = new Piece(PieceColor.WHITE);
+
+        assertTrue(this.blackPiece.isSameColorAs(blackPiece2));
+        assertTrue(this.whitePiece.isSameColorAs(whitePiece2));
+        assertFalse(blackPiece2.isSameColorAs(this.whitePiece));
+        assertFalse(whitePiece2.isSameColorAs(this.blackPiece));
+    }
+
+    @Test
+    public void testIsWhite() {
+        assertTrue(this.whitePiece.isWhite());
+        assertFalse(this.blackPiece.isWhite());
+    }
+
+    @Test
+    public void testKingMe() {
+        assertFalse(this.blackPiece.isKing());
+
+        this.blackPiece.kingMe();
+
+        assertTrue(this.blackPiece.isKing());
+    }
+}
```

Comment:
I know where you're coming from. The idea with the unit tests is to provide a foundation that protects the source code from breaking. So if someone changed the implementation of kingMe(), a test should break. Untested code is dangerous code!